test/e2e/network/dual_stack.go

```diff
@@ -118,9 +118,7 @@ var _ = common.SIGDescribe("[Feature:IPv6DualStack]", func() {
 
 		// get all schedulable nodes to determine the number of replicas for pods
 		// this is to ensure connectivity from all nodes on cluster
-		// FIXME: tests may be run in large clusters. This test is O(n^2) in the
-		// number of nodes used. It should use GetBoundedReadySchedulableNodes().
-		nodeList, err := e2enode.GetReadySchedulableNodes(ctx, cs)
+		nodeList, err := e2enode.GetBoundedReadySchedulableNodes(ctx, cs, 10)
```

Comment:
Should we do the same above on line 61? Also, why the choice of 10 nodes?

Reply:
@rjsadow 
I couldn't determine the exact size of the cluster where the test environment runs. I've tentatively set it to 10 as a starting point, but this can be adjusted based on further insights or recommendations.

Reply:
Hm... apparently I'm the one who added that comment, but I don't remember why I didn't just fix the code at the time....

I don't think there's much argument for using a large number of nodes here; we want to test that pod-to-pod traffic across nodes works, but if it works between any two randomly-selected nodes, then it most likely works across _all_ nodes. (We don't have any corresponding test for single-stack connectivity testing, so I don't see why we should need so much testing for dual-stack.)

So I think I'd go with "3".

> Should we do the same above on line 61?

No, the `It("should have ipv4 and ipv6 internal node ip"` test is fine; it just looks at every Node object, rather than actually deploying a pod to every node like this test was doing.

Reply:
@danwinship Thank you for your  feedback!! I appreciate your guidance on this matter.